docs/multisite/languages/automated_translations.md

```diff
@@ -0,0 +1,144 @@
+---
+description: With the automated translation add-on, users can translate content items into multiple languages with Google Translate or DeepL.
+---
+
+# Automated content translation
+
+The automated translation add-on package allows users have content items machine-translated into multiple languages by using either Google Translate or DeepL external translation engine.
```

Comment:
```suggestion
With the automated translation add-on package, users can translate their content items into multiple languages automatically by using either Google Translate or DeepL external translation engine.
```
In order to avoid the "allows have translated".